packages/shared/styles/components/molecules/SfSearchBar.scss

```diff
@@ -29,11 +29,18 @@
   padding: var(--search-bar-padding);
   width: var(--search-bar-width);
   height: var(--search-bar-height, 2rem);
-  &__icon {
-    --icon-size: 1.25rem;
+  &__button {
+    // --icon-size: 1.25rem;
```

Comment:
R: before merge this PR we should remove these comments ;)

Reply:
removed